### DISCUSSION TITLE
Working/Not Working Zigbee Gateways for Raspberry Pi

### DISCUSSION THREAD

**Discussion by wewa00** [diyhue/diyHue#564] [Q&A]
Hello,

is there any list with Zigbee Gateways for Raspberry Pi which are working with diyHue?

BR wewa

**Comment by mariusmotea** [comment]
Hi,

Currently Deconz and ZigBee2MQTT (lights only). In the near future ZigBee2MQTT will work also with sensors.

**Reply by wewa00** [reply]
You mean diyHue is communication with Deconz and Deconz with its supported Gateway Hardware?

**Comment by mariusmotea** [comment]
Deconz replicate the Hue Bridge and communicate directly with the lights and sensors.
